example.py

```diff
@@ -235,10 +299,9 @@ def test_run_many_remote_commands_singly():
 
 
 def test_run_many_remote_commands_serially():
-    """run a list of `remote` commands serially. `remote` commands run serially 
-    with `execute` do not share a ssh connection (at time of writing)
-    TODO: investigate above statement. explain why
-    """
+    """run a list of `remote` commands serially.
+    The `execute` module is aimed at running commands in parallel. 
+    Serial execution exists only as a sensible default and offers nothing extra."""
```

Comment:
run the two lines together, they are one statement with two sentences, not two statements.